Kickstarter-iOS/Views/Controllers/FacebookConfirmationViewController.swift

```diff
@@ -67,17 +67,17 @@ internal final class FacebookConfirmationViewController: UIViewController,
     _ = self.rootStackView |> loginRootStackViewStyle
   }
 
-    override func bindViewModel() {
+  override func bindViewModel() {
     super.bindViewModel()
 
     self.viewModel.outputs.displayEmail
-      .observeForControllerAction()
+      .observeForUI()
```

Comment:
Subtleness here prevented the email from rendering immediately. Screenshot tests were rendering the placeholder!